core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -3894,12 +3925,16 @@ public synchronized HttpRedirect doQuietDown() {
      *
      * @param block Block until the system really quiets down and no builds are running
      * @param timeout If non-zero, only block up to the specified number of milliseconds
+     * @param reason Quiet reason that will be visible to user
      */
     @RequirePOST
-    public HttpRedirect doQuietDown(@QueryParameter boolean block, @QueryParameter int timeout) throws InterruptedException, IOException {
```

Comment:
Why are we adding this method when the intended new endpoint is in the `ShutdownLink` anyway? Or is one intentionally as the UI endpoint and the other as API endpoint?